openlibrary/templates/type/edition-history/view.html

```diff
@@ -9,7 +9,7 @@
 $var title: $title
 
 $ lccn = page["lccn"] and page["lccn"][0] and ("http://lccn.loc.gov/" + page["lccn"][0])
```

Comment:
Done. Scrubbed the other common root query URLs in OL while I was at it to look for sites that fully redirect to https in openlibrary/plugins/openlibrary/pages/config_edition.page and added as appropriate. I doubt it had been done for some time.